src/test/scala/edu/berkeley/cs/rise/opaque/OpaqueOperatorTests.scala

```diff
@@ -395,6 +395,27 @@ trait OpaqueOperatorTests extends FunSuite with BeforeAndAfterAll { self =>
     df.select(year($"date")).collect
   }
 
+  testAgainstSpark("case when") { securityLevel =>
```

Comment:
I don't think this will test all of the queries, since `testAgainstSpark` will take the final returned values from this function and compare them.